sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -162,13 +165,23 @@ def create_experiment(
                     )
                 )
 
+    # fmt: off
```

Comment:
@andreyvelich Thank you for your suggestion! I have removed the code that disables formatting for the entire function. 

Is it necessary to disable formatting when checking imports? Since the import statements here have already been adjusted according to the pre-commit requirements, there are no errors here.

Reply:
Then, we should be fine, thank you!